ipatests/test_integration/test_commands.py

```diff
@@ -1292,6 +1293,45 @@ def test_pkispawn_log_is_present(self):
         assert "DEBUG" in pkispawnlog
         assert "INFO" in pkispawnlog
 
+    def test_password_lock_ldap_logs(self):
+        """
+        Test that when a user fails LDAP authentication while in lockout
+        that it is logged.
+        """
+        user = 'ldapuser'
+        original_passwd = 'Secret123'
+        new_passwd = 'newPasswd123'
+        bad_passwd = 'foo'
+        basedn = self.master.domain.basedn
+        binddn = DN(f"uid={user},cn=users,cn=accounts,{basedn}")
+
+        tasks.kinit_admin(self.master)
+        tasks.user_add(self.master, user, password=original_passwd)
+        tasks.kinit_user(
+            self.master, user,
+            '{0}\n{1}\n{1}\n'.format(original_passwd, new_passwd)
+        )
+
```

Comment:
Nitpick: there is a method tasks.create_active_user that does kinit admin + user-add + kinit to set the new password.
Not a blocker, you can merge as-is.

Reply:
I'll go ahead and change it. If not now then it'll never be done. The single call is clearer.